sdk/src/test/scala/com/horizen/certificatesubmitter/CertificateSubmitterTest.scala

```diff
@@ -386,11 +387,8 @@ class CertificateSubmitterTest extends JUnitSuite with MockitoSugar {
     })
     val mockedSidechainNodeViewHolderRef: ActorRef = mockedSidechainNodeViewHolder.ref
 
-    val secureEnclaveApiClient = TestProbe()
-    secureEnclaveApiClient.setAutoPilot((sender: ActorRef, _: Any) => { sender ! None; TestActor.KeepRunning })
-
     val certificateSubmitterRef: TestActorRef[CertificateSubmitter] = TestActorRef(
-      Props(new CertificateSubmitter(mockedSettings, mockedSidechainNodeViewHolderRef, secureEnclaveApiClient.ref, params, mockedMainchainChannel)))
+      Props(new CertificateSubmitter(mockedSettings, mockedSidechainNodeViewHolderRef, mock[SecureEnclaveApiClient], params, mockedMainchainChannel)))
```

Comment:
Extend `def newBlockArrived(): Unit = {` test, case "test 2" emulate signatures from both wallet and enclave.

Reply:
I was not able to modify it to have 2 sigs in Test 2, as it keep on failing further. So I replaced one from wallet with one from enclave. Wallet sigs are tested further in a test several times, so they are covered.